pkg/controller/virtualmachine/virtualmachine_controller.go

```diff
@@ -92,52 +93,65 @@ func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	podFinalizerName := pool_manager.RuntimeObjectFinalizerName + r.poolManager.GetManagerPodName()
```

Comment:
Is there a difference between r.pool_manager and pool_manager?

I think this should be done at [NewPoolManager()](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/a8c13fea660f0298ca15b946edb0afbf1ddd1341/pkg/pool-manager/pool.go#L62), 
with string formatting while removing the `/` from RuntimeObjectFinalizerName so ti will be part of the string format
`fmt.Sprintf("%s/%s", RuntimeObjectFinalizerName, managerPodName`

Reply:
I think pool_manager library is used only for using structs/const variables introduced their, while r.pool_manager is the actual instance of the NewPoolManager() that the reconciler holds.

I agree. Will move it to the pool_manager.